### ISSUE TITLE
Different looks of tiles in the landing page and the management page

### ISSUE THREAD

**Issue by hoanphungt** [nens/lizard-management-client#942]
@joeriverheijden 

Currently, the tiles in the landing page look different from the tiles in the management page.

Remco thinks it makes our app inconsistent, more confusing and unprofessional. So either they should all use the old tiles or new tiles. It's also less work to reuse components.

Another point is that Remco just put the old tile into a library so both 3Di and Lizard can use, but now we make a new tile for the landing page.

These things could be prevented from happening from the design phase.

**Comment by hoanphungt** [comment]
@joeriverheijden @remcogerlich 
Hi Joeri,

I think Remco raised a very legitimate concern here. What is your opinion about it? And in case if we want one tile only throughout our app then which tile do you prefer?

**Comment by joeriverheijden** [comment]
Yes already discussed with Remco via Slack. Ultimately this is @Derryrover responsibility to oversee this as TechLead. 
It was mentioned a few times in earlier discussions. 

Its good to implement the new tiles in the rest of the management interface as well to make things consistent for Lizard and 3Di. 


@remcogerlich i think it's good to plan a meeting with Joep to discuss the library you're working on. Its currently quite unclear for us what implications the library has for the management interface and we would like to have a more clearer view on this. We don't want a lot of extra implentation costs that might be induced by reimplementing the library. @remcogerlich could you plan a meeting with Joep, me and Hoan to discuss this? 

Thanks!
